scripts/install_and_enable_addons_render.py

```diff
@@ -0,0 +1,519 @@
+import argparse
+import json
+import re
+import sys
+from pathlib import Path
+from random import seed
+
+import bpy
+
+
+# %% Main function
+def main():
+    """
+    User defined:
+    --seed:     Randomisaiton Seed
+    --input:    Input json file
+    --output:   Output json file
+    --render_seg_only:   Render the segmentation only
+    --render_main_only:   Render the main image only
+
+    """
+
+    # get Python args (passed after "--")
+
+    argv = sys.argv
+    if "--" not in argv:
+        argv = []
+    else:
+        argv = argv[argv.index("--") + 1 :]  # get all args after "--"
+
+    # ---------
+    # initialise parser
+    parser = argparse.ArgumentParser(
+        description=(
+            "To launch Blender and install+enable the desired add-ons, run:"
+            "  blender --python "
+            + __file__
+            + " -- [list of paths to addons to install and enable OR"
+            " path to parent dir]"
+            ""
+            "To launch Blender *with factory settings* add:"
+            "  blender --factory-startup --python "
+            + __file__
+            + " -- [list of paths to addons to install and enable]"
+        )
+    )
+
+    # add arguments
+    # required (positonal)
+    parser.add_argument(
+        "addons_paths",
+        nargs="*",
+        type=str,  # types: string, int, long, choice, float and complex.
+        metavar="ADDONS_PATHS",  # A name for the argument in usage messages.
+        help="Space-separated list of the paths to the add-ons to enable OR"
+        "path to parent dir",
+    )
+
+    parser.add_argument(
+        "-s",
+        "--seed",
+        metavar="N",
+        type=int,
+        nargs="+",
+        help="an integer for the randomisation seed",
+    )
+
+    parser.add_argument(
+        "-i",
+        "--input",
+        nargs="*",
+        type=str,  # types: string, int, long, choice, float and complex.
+        metavar="INPUT_JSON_FILE",  # A name for argument in usage messages.
+        help="Input .json file to set the min-max values for each panel",
+    )
+
+    # LB: Add frame number
+    parser.add_argument(
+        "-f",
+        "--frame",
+        nargs="*",
+        type=int,  # types: string, int, long, choice, float and complex.
+        # A name for argument in usage messages.
+        help="Input frame number to render",
+    )
+
+    # LB: Add basename
+    parser.add_argument(
+        "-b",
+        "--basename",
+        nargs="*",
+        type=str,  # types: string, int, long, choice, float and complex.
+        # A name for argument in usage messages.
+        help="Input basename number to render",
+    )
+
+    # LB: Segmentation only
+    # Add the flag for render_main_only
+    parser.add_argument(
+        "-rm",
+        "--render_main",
+        nargs="*",
+        type=str,  # types: string, int, long, choice, float and complex.
+        # A name for argument in usage messages.
+        help="Input bool to render",
```

Comment:
I don't think this help is clearly describing what this argument does. Can it be rephrased?